tests/agent/actions/test_list_table_names_action.py

```diff
@@ -31,11 +32,30 @@ def dummy_org_id() -> str:
     return "94af8c91-6e56-4f2a-9b1f-04868321c5f5"
 
 
+@pytest.fixture
+def dummy_access_token() -> str:
+    return (
+        "5a43e329ddd64ca286bf58574d17f9e1.V1"
+        ".myBie29LH0m_5CTw0RCeAtBXWb5V519lAeIq1rF4WWN4WZrJGMe0GAMcnuuwYkveR0ptvUFoeeK2zCt6NJMiSg"
+    )
+
+
+@pytest.fixture
+def mock_gx_agent_env_vars(dummy_base_url, dummy_org_id, dummy_access_token) -> None:
+    with mock.patch(
+        "great_expectations.agent.actions.list_table_names.GxAgentEnvVars"
+    ) as mock_env_vars:
+        mock_env_vars.return_value = GxAgentEnvVars(
+            gx_cloud_base_url=dummy_base_url,
+            gx_cloud_organization_id=dummy_org_id,
+            gx_cloud_access_token=dummy_access_token,
+        )
+        yield
```

Comment:
looks like there's a mypy issue with this - you might consider either using the same pattern as [this fixture](https://github.com/great-expectations/great_expectations/blob/38aebde90990e9f57fe8aec89d5df3e998f74f09/tests/agent/actions/test_draft_datasource_config.py#L31), or just moving it up to a common conftest.

Reply:
Thanks for the rec - addressed in https://github.com/great-expectations/great_expectations/pull/8450/commits/4d3629ad95163869f3935fffd02821249176dbbd